connmgr/connmanager.go

```diff
@@ -342,3 +342,15 @@ func New(cfg *Config) (*ConnManager, error) {
 	}
 	return &cm, nil
 }
+
+// TstSetMaxRetryDuration makes the ability to set the max retry duration
+// available to the test package.
+func TstSetMaxRetryDuration(d time.Duration) {
```

Comment:
This function isn't necessary anymore since the tests are in the same package.  They can just do `maxRetryDuration = whatever`.